test/View/RouteNotFoundStrategyTest.php

```diff
@@ -16,9 +16,12 @@
 use Zend\Mvc\MvcEvent;
 use Zend\Mvc\View\Http\RouteNotFoundStrategy;
 use Zend\View\Model\ViewModel;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class RouteNotFoundStrategyTest extends TestCase
 {
+    use EventManagerIntrospectionTrait;
```

Comment:
Doing so makes the verbiage far more verbose, though:

```php
$this->getListenersForEvent(...);

// vs
EventManagerIntrospection::getListenersForEvent(...)
```

Let's leaves as-is, with the improvements and additions you've suggested.
I'll get those pushed shortly.
On Oct 1, 2015 7:36 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In test/View/RouteNotFoundStrategyTest.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40907654>:
>
> >
> >  class RouteNotFoundStrategyTest extends TestCase
> >  {
> > +    use EventManagerIntrospectionTrait;
>
> As mentioned above, you are copying the helper methods into every test
> case, instead of just having them available in the test suite.
>
> Since they are pure helper methods, you can just have them as static
> methods in a separate class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40907654>.
>